flight/targets/sparky/fw/pios_board.c

```diff
@@ -6,7 +6,8 @@
  * @{
  *
  * @file       pios_board.c
- * @author     Tau Labs, http://taulabs.org, Copyright (C) 2012-2014
+ * @author     dRonin, http://dronin.org Copyright (C) 2015
```

Comment:
https://github.com/peabody124/TauLabs/commits/copyrights/flight/targets/sparky/fw/pios_board.c shows that this doesn't have any recent changes. Included by accident?